src/components/reactions/Like.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import styled, { css } from "styled-components";
+
+import { ReactComponent as likeIcon } from "../../assets/svg/icons/thumbs-up.svg";
+
+import "./Like.css";
+
+function Like({ counts }) {
+  const [isReacted, setIsReacted] = useState(false);
+
+  const handleClick = () => {
+    setIsReacted(!isReacted);
+  };
```

Comment:
callback 
리액트에서는 `updater function`라고 표현한다
```suggestion
    setIsReacted((isReacted) => !isReacted);
  };
```